### ISSUE TITLE
Temporal styloid process elongated 

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10148]
**Preferred term label:**
Temporal styloid process elongated 

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C5435468

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Eagle syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000044
- label: Elongated temporal styloid process
- parents: [Abnormal temporal bone morphology (HP:0009911)](https://hpo.jax.org/app/browse/term/HP:0009911)
- pmid: PMID:31082019
- def: The length of the temporal styloid process is above the upper limit of normal. The styloid process is a cylindrical, slender, needle-like projection of varying lengths averaging 2 to 3 cm. The styloid process projects from the inferior part of the petrous temporal bone and offers attachment to the stylohyoid ligament and the stylohyoid, stylopharyngeus, and styloglossus muscles.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10148
- synonyms: Temporal styloid process elongated(NONE)
